<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CddbSearchDialog</class>
 <widget class="QDialog" name="CddbSearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CDDB Search</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabSearchMethod">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Automatic</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Search for album ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelDiscId">
         <property name="text">
          <string>--</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonSearchA">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Search</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Manual</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Words</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineSearchString"/>
       </item>
       <item>
        <widget class="QCheckBox" name="buttonSearchAll">
         <property name="text">
          <string>All fields</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string>groupSearchFieldButtons</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="buttonSearchArtist">
         <property name="text">
          <string>Artist</string>
         </property>
         <attribute name="buttonGroup">
          <string>groupSearchFieldButtons</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="buttonSearchAlbum">
         <property name="text">
          <string>Album</string>
         </property>
         <attribute name="buttonGroup">
          <string>groupSearchFieldButtons</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="buttonSearchTitle">
         <property name="text">
          <string>Track</string>
         </property>
         <attribute name="buttonGroup">
          <string>groupSearchFieldButtons</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonSearchM">
         <property name="text">
          <string>Search</string>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableView" name="tableSearchResults">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>2</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="baseSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>24</number>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTableView" name="tableTrackListing">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>2</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>24</number>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="leftMargin">
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Tags</string>
       </property>
       <property name="indent">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonAllTags">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>All tags</string>
       </property>
       <property name="statusTip">
        <string>Apply all tags</string>
       </property>
       <property name="text">
        <string>All</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string>groupTagButtons</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonArtistTag">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Artist</string>
       </property>
       <property name="statusTip">
        <string>Apply artist name</string>
       </property>
       <property name="text">
        <string>A</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string>groupTagButtons</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonAlbumTag">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Album</string>
       </property>
       <property name="statusTip">
        <string>Apply album name</string>
       </property>
       <property name="text">
        <string>B</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string>groupTagButtons</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonGenreTag">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Genre</string>
       </property>
       <property name="statusTip">
        <string>Apply genre</string>
       </property>
       <property name="text">
        <string>G</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string>groupTagButtons</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonNumberTag">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Track number</string>
       </property>
       <property name="statusTip">
        <string>Apply track number</string>
       </property>
       <property name="text">
        <string>N</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string>groupTagButtons</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonMaxNumberTag">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string extracomment="Track count">Disc's track count</string>
       </property>
       <property name="statusTip">
        <string>Apply track count</string>
       </property>
       <property name="text">
        <string>M</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string>groupTagButtons</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonTitleTag">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Track Title</string>
       </property>
       <property name="statusTip">
        <string>Apply track title</string>
       </property>
       <property name="text">
        <string>T</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string>groupTagButtons</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonYearTag">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string extracomment="Year">Year</string>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="text">
        <string>Y</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string>groupTagButtons</string>
       </attribute>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="checkMultiArtist">
       <property name="toolTip">
        <string>Mark album as having many artists</string>
       </property>
       <property name="statusTip">
        <string>Mark album as having many artists</string>
       </property>
       <property name="text">
        <string>Multi-artist</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maximum">
        <number>0</number>
       </property>
       <property name="value">
        <number>-1</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelInfo">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Downloading...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonsDialog">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Apply|QDialogButtonBox::Close</set>
       </property>
       <property name="centerButtons">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonsDialog</sender>
   <signal>accepted()</signal>
   <receiver>CddbSearchDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonsDialog</sender>
   <signal>rejected()</signal>
   <receiver>CddbSearchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="groupTagButtons">
   <property name="exclusive">
    <bool>true</bool>
   </property>
  </buttongroup>
  <buttongroup name="groupSearchFieldButtons"/>
 </buttongroups>
</ui>
